churaverse-engine-client/src/eventbus/IEventBus.ts

```diff
@@ -20,6 +20,15 @@ export interface IEventBus<Scene extends Scenes> {
     priority?: EventPriority
   ) => void
 
+  /**
+   * イベントがpostされた時に実行されるlistener(callback)を解除する.
+   * @param type イベント名
+   * @param listener typeで指定したイベントがpostされた時に実行される関数
```

Comment:
特にこだわりがなければ、frontendとbackendで説明をどっちかに揃えた方が良いですね

```suggestion
   * イベントがpostされた時に実行されるlistener(callback)を削除する.
   * @param type イベント名
   * @param listener 削除するlistener
```